src/storage/mvcc/reader.rs

```diff
@@ -194,6 +197,19 @@ impl<'a> MvccReader<'a> {
                 }
             }
         }
+        Ok(Some(ts))
+    }
+
+    pub fn get(&mut self, key: &Key, mut ts: u64) -> Result<Option<Value>> {
+        // Check for locks that signal concurrent writes.
+        match self.isolation_level {
+            IsolationLevel::SI => {
```

Comment:
I prefer to use `if  self.isolation_leve==  IsolationLevel::SI` instead of `match` here?